packages/react-native-renderer/src/ReactNativeFiberRenderer.js

```diff
@@ -155,12 +176,15 @@ const NativeRenderer = ReactFiberReconciler({
     return false;
   },
 
-  getRootHostContext(): {} {
-    return emptyObject;
+  getRootHostContext(rootContainerInstance: Container): HostContext {
+    return {type: 'root'};
   },
 
-  getChildHostContext(): {} {
-    return emptyObject;
+  getChildHostContext(
+    parentHostContext: HostContext,
+    type: string,
+  ): HostContext {
+    return {type};
```

Comment:
This is providing a new context object for every node in the tree. We should only do this in DEV and even in DEV we should only do this when we are switching between useful context differences (e.g. from non-text to text). In DOM we change the context only when we switch between HTML and SVG, not every node along the way.

Reply:
Yah, hadn't yet pushed the `DEV` only check, but it's there now.